src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -460,6 +460,26 @@ static Library()
                     returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
                     targetFunction: SingleArgTrig(x => x * 180.0 / Math.PI))
             },
+            {
+                BuiltinFunctionsCore.Dec2Hex,
+                StandardErrorHandling<NumberValue>(
```

Comment:
>StandardErrorHandling [](http://example.com/codeflow?start=16&length=21)

Need to pass the function name as the first argument of StandardErrorHandling<T> now. In this case, it would be BuiltinFunctionsCore.Dec2Hex.Name